<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#LoginWindow{background-color: qlineargradient(spread:pad, x1:0.512, y1:1, x2:0.508, y2:0, stop:0.0696517 rgba(70, 125, 120, 255), stop:0.890547 rgba(255, 255, 255, 255));}
#stackedWidget{ background-color: rgba(1,1,1,0);}

#LoginButton{ background-color: qlineargradient(spread:pad, x1:0.512, y1:1, x2:0.503, y2:0.306727, stop:0.0696517 rgba(158, 246, 169, 255), stop:0.890547 rgba(255, 255, 255, 255));}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLineEdit" name="UsernameInputBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PasswordInputBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ClearButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>254</y>
        <width>89</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>AnjaliOldLipi</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">/*border-top-color: rgb(85, 87, 83);*/</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RegisterButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>254</y>
        <width>89</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>AnjaliOldLipi</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ExitButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>89</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>AnjaliOldLipi</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="LoginButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>310</y>
        <width>89</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.512000000000000" starty="1.000000000000000" endx="0.503000000000000" endy="0.306727000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.069651700000000">
             <color alpha="255">
              <red>158</red>
              <green>246</green>
              <blue>169</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.890547000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.512000000000000" starty="1.000000000000000" endx="0.503000000000000" endy="0.306727000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.069651700000000">
             <color alpha="255">
              <red>158</red>
              <green>246</green>
              <blue>169</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.890547000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.512000000000000" starty="1.000000000000000" endx="0.503000000000000" endy="0.306727000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.069651700000000">
             <color alpha="255">
              <red>158</red>
              <green>246</green>
              <blue>169</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.890547000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.512000000000000" starty="1.000000000000000" endx="0.503000000000000" endy="0.306727000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.069651700000000">
             <color alpha="255">
              <red>158</red>
              <green>246</green>
              <blue>169</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.890547000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.512000000000000" starty="1.000000000000000" endx="0.503000000000000" endy="0.306727000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.069651700000000">
             <color alpha="255">
              <red>158</red>
              <green>246</green>
              <blue>169</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.890547000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.512000000000000" starty="1.000000000000000" endx="0.503000000000000" endy="0.306727000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.069651700000000">
             <color alpha="255">
              <red>158</red>
              <green>246</green>
              <blue>169</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.890547000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.512000000000000" starty="1.000000000000000" endx="0.503000000000000" endy="0.306727000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.069651700000000">
             <color alpha="255">
              <red>158</red>
              <green>246</green>
              <blue>169</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.890547000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.512000000000000" starty="1.000000000000000" endx="0.503000000000000" endy="0.306727000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.069651700000000">
             <color alpha="255">
              <red>158</red>
              <green>246</green>
              <blue>169</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.890547000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="LinearGradientPattern">
           <gradient startx="0.512000000000000" starty="1.000000000000000" endx="0.503000000000000" endy="0.306727000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
            <gradientstop position="0.069651700000000">
             <color alpha="255">
              <red>158</red>
              <green>246</green>
              <blue>169</blue>
             </color>
            </gradientstop>
            <gradientstop position="0.890547000000000">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </gradientstop>
           </gradient>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>AnjaliOldLipi</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="UsernameLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>120</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AnjaliOldLipi</family>
       </font>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="PasswordLabel">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AnjaliOldLipi</family>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AnjaliOldLipi</family>
        <pointsize>25</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/*#label {font-size: 25px;}*/</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
